src/GenFu/GenFu.cs

```diff
@@ -76,9 +76,9 @@ public static List<object> ListOf(Type type)
             return BuildList(type, _listCount);
         }
         
-        public static List<T> ListOf<T>(int personCount) where T : new()
+        public static List<T> ListOf<T>(int itemCount) where T : new()
         {
-            return ListOf(typeof(T), personCount).Cast<T>().ToList();
+            return ListOf(typeof(T), itemCount).Cast<T>().ToList();
         }
 
         public static List<object> ListOf(Type type, int personCount)
```

Comment:
Hmmm....looks like there's another variable just below with the same naming issue. Would you mind fixing this, too?